sematic/db/models/external_resource.py

```diff
@@ -0,0 +1,192 @@
+# Standard Library
+import datetime
+from typing import Any, Dict, Tuple, Type, Union
+
+# Third-party
+from sqlalchemy import Column, types
+from sqlalchemy.orm import validates
+
+# Sematic
+from sematic.db.models.base import Base
+from sematic.db.models.json_encodable_mixin import JSONEncodableMixin
+from sematic.external_resource import ExternalResource as ExternalResourceDataclass
+from sematic.external_resource import ManagedBy, ResourceState
+from sematic.types.serialization import (
+    type_from_json_encodable,
+    type_to_json_encodable,
+    value_from_json_encodable,
+    value_to_json_encodable,
+)
+
+TypeSerialization = Dict[str, Any]
+
+
+class ExternalResource(Base, JSONEncodableMixin):
+    """A DB record for an ExternalResource (dataclass) and its history.
+
+    For the remainder of this docstring, ExternalResource will correspond
+    to the non-orm representation (aka the dataclass representation).
+
+    Attributes
```

Comment:
Please add the new `created_at` and `updated_at` fields docstrings.

Reply:
done